TopMovies/View/MovieCollectionViewCell.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  MovieCollectionViewCell.swift
+//  TopMovies
+//
+//  Created by anikolaenko on 08.02.2021.
+//
+
+import UIKit
+import SnapKit
+import Nuke
+
+class MovieCollectionViewCell: UICollectionViewCell {
+    private let posterImage = UIImageView()
+    private let adultLabel = UILabel()
+    private let ratingLabel = UILabel()
+    private let titleLabel = UILabel()
+    private let descriptionLabel = UILabel()
+    private let shortInfoStackView = UIStackView()
+    private let infoStackView = UIStackView()
+    
+    public func configureWith(movie: Movie) {
+        if let url = movie.poster { Nuke.loadImage(with: url, into: posterImage) }
+        adultLabel.text = "Adult: \(movie.adult ? "Yes" : "No")"
+        ratingLabel.text = String(movie.rating)
+        titleLabel.text = movie.title
+        descriptionLabel.text = movie.description
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        setupViewHierarchy()
+        setupLayout()
+        setupStyle()
+    }
```

Comment:
You shouldn't call those methods in that` init` since the view wouldn't be loaded here.